railties/test/generators/controller_generator_test.rb

```diff
@@ -82,6 +82,16 @@ def test_skip_routes_prevents_generating_tests_with_routes
     end
   end
 
+  def test_does_not_write_multiple_times_same_route
```

Comment:
This test should probably go in actions_test.rb.  See: https://github.com/rails/rails/blob/f78fb42b51c2226b5e87d5c637604f1d85b68b14/railties/test/generators/actions_test.rb#L503-L516

Reply:
Thanks.